### PR TITLE
kcptun v20200103 (new formula)

### PR DESCRIPTION
**Pull request by gaoyifan**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by gaoyifan** [comment]
Hey guys, anything need to change? 😃

File: Formula/kcptun.rb [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-ldflags", "-X main.VERSION=#{version} -s -w",
+      "-o", bin/"kcptun_client", "github.com/xtaci/kcptun/client"
+    system "go", "build", "-ldflags", "-X main.VERSION=#{version} -s -w",
+      "-o", bin/"kcptun_server", "github.com/xtaci/kcptun/server"
+
+    (buildpath/"kcptun_client.json").write <<~EOS
```
Comment (SMillerDev) [inline Formula/kcptun.rb]: This needs to be provided by upstream or not at all.

**Comment by SMillerDev** [comment]
Thanks @gaoyifan! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by chenrui333** [comment]
Thanks @gaoyifan!

**Pull request closed without merging**